src/ompl/base/objectives/src/MaximizeMinClearanceObjective.cpp

```diff
@@ -50,9 +50,14 @@ ompl::base::Cost ompl::base::MaximizeMinClearanceObjective::stateCost(const Stat
     return Cost(si_->getStateValidityChecker()->clearance(s));
 }
 
+bool ompl::base::MaximizeMinClearanceObjective::isSatisfied(Cost c) const
+{
+    return isCostBetterThan(c, Cost(threshold_.value() + magic::BETTER_PATH_COST_MARGIN));
```

Comment:
I agree. Looking through this, I am thinking we should just get rid of this margin completely. Thoughts?